CONTRIBUTING.md

```diff
@@ -6,8 +6,7 @@ We appreciate your help!
 
 ## Before filing an issue
 
-If you are unsure whether you have found a bug, please consider asking in the [vald slack](https://vald-community.slack.com/messages/CN1RDC8NB) first. If
-the behavior you are seeing is confirmed as a bug or issue, it can easily be re-raised in the issue tracker.
+If you are unsure whether you have found a bug, please consider asking in the [vald slack](https://vald-community.slack.com/messages/CN1RDC8NB) first. If the behavior you are seeing is confirmed as a bug or issue, it can easily be re-raised in the issue tracker.
 
 ## Filing issues
 
```

Comment:
![badge](https://img.shields.io/badge/review-imo-orange.svg?style=flat-square)
The link in L13 should also be the invitation link. and notated as "the security channel in Vald Slack".